internal/storage/storage.go

```diff
@@ -92,7 +90,7 @@ func (t *Storage) GetTopic(id int) (*models.Topic, error) {
 func (t *Storage) GetRecentTopics() ([]models.Topic, error) {
 	topics := []models.Topic{}
 	query := `
-		SELECT DISTINCT topics.*,
+		SELECT DISTINCT topics.id, topics.title,
```

Comment:
Is there a way to load these from external file on load?

Reply:
Maybe the `Storage` struct could be updated to have a `.sql` field that is a map of SQL files parsed as strings. Then the `New` function in this file could parse SQL files at creation of a `Storage` object and populate the map. Then you could use it like this:

`rows, err := t.db.Query(t.db.SQL["GetRecentTopics"])`

I think this approach might get a little annoying for SQL that includes parameters though, because it would work like this:

`rows, err := t.db.Query(t.db.SQL["my_query"], userID, limit)`

But since the `my_query` SQL is in another file I'd need to check that file to know it needs the `userID` and `limit` params. And I wouldn't get any type of autocomplete since it's not a formal method on a struct or anything.

I'll have to think about this one some more. I'm sure there are solutions out there.

Reply:
Cool. Not a blocker, you're doing good work!

Reply:
Thanks! 🙌